ospfd/ospf_vty.c

```diff
@@ -12952,6 +12952,42 @@ DEFUN (clear_ip_ospf_interface,
 	return CMD_SUCCESS;
 }
 
+DEFPY_HIDDEN(ospf_lsa_refresh_timer, ospf_lsa_refresh_timer_cmd,
```

Comment:
I have the same question, is it supposed to be hidden from the operator (and not documented) or not?

Reply:
Yes this cli needs to be hidden from then operator.